src/eth_rpc/servers/debug_rpc.rs

```diff
@@ -60,7 +59,27 @@ impl<P: EthereumProvider + Send + Sync + 'static> DebugApiServer for DebugRpc<P>
     }
 
     /// Returns an array of EIP-2718 binary-encoded receipts.
-    async fn raw_receipts(&self, _block_id: BlockId) -> Result<Vec<Bytes>> {
-        Err(EthApiError::Unsupported("debug_rawReceipts").into())
+    async fn raw_receipts(&self, block_id: BlockId) -> Result<Vec<Bytes>> {
+        Ok(self
+            .eth_provider
+            .block_receipts(Some(block_id))
+            .await?
+            .unwrap_or_default()
+            .into_iter()
+            .map(|receipt| {
+                Receipt {
+                    tx_type: receipt.transaction_type.to::<u8>().try_into().unwrap(),
+                    success: receipt.status_code.unwrap_or_default().to::<u8>() == 1,
```

Comment:
the `to` method can panic, better to either use `TryInto::<u8>` or there is also a `saturating_to`

Reply:
Here as `status_code` is a `U64` I simply converted it with the to method to `u64`, which is safe. In any case, if the conversion does not give 1 it is ultimately false.